app/controllers/Subscriptions.scala

```diff
@@ -139,7 +142,17 @@ class Subscriptions(
     val css = "paperSubscriptionLandingPage.css"
     val canonicalLink = Some(buildCanonicalPaperSubscriptionLink())
 
-    Ok(views.html.main(title, id, js, css, None, canonicalLink)).withSettingsSurrogateKey
+    def pricesToMap(prices: List[ZuoraCatalogPricePlan], prefix: String) = {
+      prices.map(price => (
+        s"$prefix-${price.name.getOrElse(price.id).toLowerCase}",
+        price.pricePerPeriod.filter(_.currency == "GBP").map(_.price).sum.toString
+      )).toMap
+    }
+
+    val collectionPrices = pricesToMap(getPaperPrices.collection, "collection")
+    val deliveryPrices = pricesToMap(getPaperPrices.delivery, "delivery")
+
+    Ok(views.html.main(title, id, js, css, None, canonicalLink, data = collectionPrices ++ deliveryPrices)).withSettingsSurrogateKey
```

Comment:
Not sure if we want to send just the prices or all the plan info so we can be a bit more clever instead of hardcoding the plans in both js and scala. I believe @paulbrown1982 has some thoughts on how to no hardcode them on the scala side (email incoming) and he's super right. Might be good to use this PR to refine our fetching of prices

Reply:
I'm looking into how to handle prices at the moment, let's go with this for now and I'll rework it later